borsh-derive-internal/src/attribute_helpers.rs

```diff
@@ -1,34 +1,22 @@
-use quote::ToTokens;
-use syn::spanned::Spanned;
-use syn::{Attribute, Error, Meta, NestedMeta, Path};
+use syn::{Attribute, Path};
 
 pub fn contains_skip(attrs: &[Attribute]) -> bool {
-    for attr in attrs.iter() {
-        if let Ok(Meta::Path(path)) = attr.parse_meta() {
-            if path.to_token_stream().to_string().as_str() == "borsh_skip" {
-                return true;
-            }
-        }
-    }
-    false
+    attrs
+        .iter()
+        .filter(|attr| attr.path().is_ident("borsh_skip"))
+        .count()
+        > 0
```

Comment:
https://doc.rust-lang.org/stable/std/iter/trait.Iterator.html#method.any

```suggestion
        .any(|attr| attr.path().is_ident("borsh_skip"))
```